zookeeper/Dockerfile

```diff
@@ -49,14 +71,56 @@ COPY shared/log4shell.sh /bin
 RUN /bin/log4shell.sh /stackable/apache-zookeeper-${PRODUCT}-bin
 
 # Ensure no vulnerable files are left over
-# This will currently report vulnerable files being present, as it also alerts on
-# SocketNode.class, which we do not remove with our scripts.
-# Further investigation will be needed whether this should also be removed.
+# This will currently report vulnerable files being present, as it also alerts
+# on SocketNode.class, which we do not remove with our scripts. Further
+# investigation will be needed whether this should also be removed.
 COPY shared/log4shell_1.6.1-log4shell_Linux_x86_64 /bin/log4shell_scanner_x86_64
 COPY shared/log4shell_1.6.1-log4shell_Linux_aarch64 /bin/log4shell_scanner_aarch64
 COPY shared/log4shell_scanner /bin/log4shell_scanner
 RUN /bin/log4shell_scanner s /stackable/apache-zookeeper-${PRODUCT}-bin
 # ===
 
+# END ZOOKEEPER BUILDER
+# ==============================================================================
+
+# ==============================================================================
+# START FINAL ZOOKEEPER IMAGE
+
+FROM stackable/image/java-base
+
+ARG PRODUCT
+ARG RELEASE
+
+LABEL name="Apache ZooKeeper" \
+      maintainer="info@stackable.tech" \
+      vendor="Stackable GmbH" \
+      version="${PRODUCT}" \
+      release="${RELEASE}" \
+      summary="The Stackable image for Apache ZooKeeper." \
+      description="This image is deployed by the Stackable Operator for Apache ZooKeeper."
+
+RUN microdnf update && \
+    microdnf install tar gzip zip && \
```

Comment:
Multiline please and what do we need these for?

Reply:
They are left-overs from the previous (non-multi-stage) Dockerfile. Will remove them and test if the build succeeds.

Reply:
Removed in 94d92b6

Reply:
IIRC without them we cannot copy from or to the pod via kubectl.

Reply:
I believe we have that in the base image